setup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+
```

Comment:
please keep the `#!/bin/bash` its common practice and may cause issues if its missing in some cases